collector/src/output_frontend.cpp

```diff
@@ -426,10 +438,20 @@ void CMonitorOutputFrontend::push_current_sections_to_json(bool is_header)
         if (sec.m_measurements.empty()) {
             for (size_t subsec_idx = 0; subsec_idx < sec.m_subsections.size(); subsec_idx++) {
                 auto& subsec = sec.m_subsections[subsec_idx];
-
-                push_json_object_start(subsec.m_name, THIRD_LEVEL);
-                push_json_measurements(subsec.m_measurements, FOURTH_LEVEL);
-                push_json_object_end(subsec_idx == sec.m_subsections.size() - 1, THIRD_LEVEL);
+                if (subsec.m_measurements.empty()) {
+                    push_json_object_start(subsec.m_name, SECOND_LEVEL);
```

Comment:
I think this should be a THIRD_LEVEL instead of SECOND_LEVEL here... and then later on we should increase all by one and reach out to FIFTH_LEVEL for indentation right?

Reply:
@f18m ,
You are right..!!
I have corrected it and updated the JSONS.